### ISSUE TITLE
Investigate Block Kit Builder

### ISSUE THREAD

**Issue by alysivji** [busy-beaver-dev/busy-beaver#153]
As a developer, I want to be able to design attractive Slack interfaces without a lot of effort.

Not sure what the best way forward is, but the Block Kit Builder seems promising

https://api.slack.com/tools/block-kit-builder

**Comment by alysivji** [comment]
Creating a link button. Does this send information back when clicked? If so, handle by sending back a 200 ack.

https://api.slack.com/reference/messaging/block-elements#button

```json
{
  "type": "button",
  "text": {
    "type": "plain_text",
    "text": "Link Button"
  },
  "url": "https://api.slack.com/block-kit"
}
```

**Comment by alysivji** [comment]
Closed via #154; created a simple wrapper around Slack Block Kit
